src/realm-settings/EmailTab.tsx

```diff
@@ -0,0 +1,305 @@
+import React, { useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { Controller, useForm } from "react-hook-form";
+import { useErrorHandler } from "react-error-boundary";
+import {
+  ActionGroup,
+  AlertVariant,
+  Button,
+  Checkbox,
+  FormGroup,
+  PageSection,
+  Switch,
+  TextInput,
+} from "@patternfly/react-core";
+
+import RealmRepresentation from "keycloak-admin/lib/defs/realmRepresentation";
+import { useAdminClient, asyncStateFetch } from "../context/auth/AdminClient";
+import { useRealm } from "../context/realm-context/RealmContext";
+import { useAlerts } from "../components/alert/Alerts";
+import { FormAccess } from "../components/form-access/FormAccess";
+import { HelpItem } from "../components/help-enabler/HelpItem";
+import { FormPanel } from "../components/scroll-form/FormPanel";
+
+import "./RealmSettingsSection.css";
+
+export const RealmSettingsEmailTab = () => {
+  const { t } = useTranslation("realm-settings");
+  const adminClient = useAdminClient();
+  const handleError = useErrorHandler();
+  const { realm: realmName } = useRealm();
+  const { addAlert } = useAlerts();
+  const { register, control, setValue, handleSubmit } = useForm();
+  const [realm, setRealm] = useState<RealmRepresentation>();
+  const [isAuthenticationEnabled, setAuthenticationEnabled] = useState("");
+
+  const form = useForm();
+
+  useEffect(() => {
+    return asyncStateFetch(
+      () => adminClient.realms.findOne({ realm: realmName }),
+      (realm) => {
+        setRealm(realm);
+        setupForm(realm);
+
+        setAuthenticationEnabled(realm?.attributes!.authentication);
+      },
+      handleError
+    );
+  }, []);
+
+  useEffect(() => {
+    setValue("attributes.loginUsername", realm?.attributes!.loginUsername);
+    setValue("attributes.loginPassword", realm?.attributes!.loginPassword);
+  }, [isAuthenticationEnabled]);
+
+  const setupForm = (realm: RealmRepresentation) => {
+    const { ...formValues } = realm;
+
+    form.reset(formValues);
+
+    Object.entries(realm).map((entry) => {
+      setValue(entry[0], entry[1]);
+    });
```

Comment:
You could destructure this array to make the code a bit more readable.
```suggestion
    Object.entries(realm).map(([key, value]) => {
      setValue(key, value);
    });
```